chart/values.yaml

```diff
@@ -154,6 +154,10 @@ agents:
         cpu: "500m"
         # -- Memory requests for core agents
         memory: "32Mi"
+    # -- How long core agent waits for a faulted child's replica to come back online so that
+    # the agent can attempt to make same child online and in turn utilize fast rebuild capability.
+    # A blank value "" means internally derived value will be used.
```

Comment:
I think we should reword this, a user doesn't know what a child is and doesn't know what child_online rpc is.
I suggest:
```suggestion
# -- If a faulted replica comes back online within this time period then it will be rebuilt using the partial rebuild capability (using a log of missed IO). Otherwise it will be fully rebuilt.
     # A blank value "" means an internally derived value will be used.
```

Reply:
Sounds good. Will modify the documentation.